examples/credential.rs

```diff
@@ -2,7 +2,7 @@
 // SPDX-License-Identifier: Apache-2.0
 
 //! A basic example that generates and publishes subject and issuer DID
-//! Documents, creates a VerifiableCredential specifying claims about the
+//! Documents, creates a Credential specifying claims about the
```

Comment:
```suggestion
//! Documents, creates a Verifiable Credential specifying claims about the
```

I think it's better to use the W3C terminology here